.prettierrc.js

```diff
@@ -3,12 +3,31 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
+/** @type {import("prettier").Config} */
 module.exports = {
   'trailingComma': 'none',
   'tabWidth': 2,
   'semi': false,
   'singleQuote': true,
   'jsxSingleQuote': true,
   'bracketSameLine': false,
-  'quoteProps': 'preserve'
+  'quoteProps': 'preserve',
+  'arrowParens': 'always',
+  'bracketSpacing': true,
+  'embeddedLanguageFormatting': 'auto',
+  'endOfLine': 'lf',
+  'htmlWhitespaceSensitivity': 'css',
+  'insertPragma': false,
+  'plugins': [],
+  'pluginSearchDirs': false,
+  'printWidth': 80,
+  'proseWrap': 'always',
+  'rangeEnd': Infinity,
+  'rangeStart': 0,
+  'requirePragma': false,
+  'useTabs': false,
+  'vueIndentScriptAndStyle': false,
+  'jsxBracketSameLine': false,
+  'singleAttributePerLine': true,
+  'overrides': undefined
```

Comment:
many of these are the same as the default, e.g. `printWidth`. Some are not used, e.g. `vueIndentScriptAndStyle`. Is there a purpose of including every configuration possible at the the current version of prettier?

Reply:
To avoid conflicts with IDE prefference